tasks/miarecweb.yml

```diff
@@ -175,6 +175,8 @@
       line: 'miarec.jobs_log_directory = {{ miarecweb_log_dir }}/jobs'
     - regexp: '^deform_mako.module_directory = '
       line: 'deform_mako.module_directory = {{ deploy_helper.new_release_path }}/cache'
+    - regexp: '^miarec.sox_path = /usr/local/bin/sox'

```

Comment:
This line should be:

```
- regexp: '^miarec.sox_path = '
```

Otherwise, we will not be able to change more than once.
For example, on the first Ansible run, we change it to `/somepath/sox`. Such a change will be successful.
Later, we decide to change it to `/otherpath/sox`. We change the corresponding variable and run ansible playbook.
The config file will not be updated.

Interestingly, such issue will occur only when the miarecweb version stays the same between ansible runs.
If we bump up the version of miareweb, the INI file will be recreated with default sox_path `/usr/local/bin/sox`.